README.md

```diff
@@ -1,18 +1,15 @@
-Windows Azure Active Directory IdentityModel Extensions for .Net
-===========
+# Windows Azure Active Directory IdentityModel Extensions for .Net
```

Comment:
Unrelated, but "Windows" is a little misleading here isn't it?

Reply:
Agreed. Suggesting that we address it during the _repo beautification_ effort.

`Microsoft IdentityModel for .NET` might be a better name.